components/infobox/commons/infobox_patch.lua

```diff
@@ -0,0 +1,89 @@
+local Class = require('Module:Class')
```

Comment:
This whole file seems to use spaces instead of tabs

Reply:
don't all the infobox ones do?

Reply:
Well that's a mistake then 😉 

Since this is a new file could you make sure it uses tabs?